esx_service/cli/vmdkops_admin_test.py

```diff
@@ -735,30 +735,70 @@ def test_tenant_access(self):
         # change total_volume size to 2GB
         volume_maxsize_in_MB = convert.convert_to_MB("1000MB")
         volume_totalsize_in_MB = convert.convert_to_MB("3GB")
-        error_info = auth_api._tenant_access_set(name=self.tenant1_name,
-                                                 datastore=self.datastore_name,
-                                                 allow_create="True",
-                                                 volume_maxsize_in_MB=volume_maxsize_in_MB,
-                                                 volume_totalsize_in_MB=volume_totalsize_in_MB
-                                             )
-        self.assertEqual(None, error_info)
 
-        error_info, privileges = auth_api._tenant_access_ls(self.tenant1_name)
-        self.assertEqual(None, error_info)
+        self.parser = vmdkops_admin.create_parser()
 
-        rows = vmdkops_admin.generate_tenant_access_ls_rows(privileges)
-        self.assertEqual(len(rows), 1)
+        for val in ["False", "FALSE", "false"]:
+            command = ("tenant access set --name={0} ".format(self.tenant1_name))
+            command += ("--datastore={0} ".format(self.datastore_name))
+            command += ("--allow-create={0} ".format(val))
+            command += ("--volume-maxsize=500MB --volume-totalsize=1GB")
 
+            args = self.parser.parse_args(command.split())
+            error_info = vmdkops_admin.tenant_access_set(args)
+            self.assertEqual(None, error_info)
 
-        expected_output = [self.datastore_name,
-                           "True",
-                           "1000.00MB",
-                           "3.00GB"]
-        actual_output = [rows[0][0],
-                         rows[0][1],
-                         rows[0][2],
-                         rows[0][3]]
-        self.assertEqual(expected_output, actual_output)
+            error_info, privileges = auth_api._tenant_access_ls(self.tenant1_name)
+            self.assertEqual(None, error_info)
+
+            rows = vmdkops_admin.generate_tenant_access_ls_rows(privileges)
+            self.assertEqual(len(rows), 1)
+
+            expected_output = [self.datastore_name,
+                            "False",
+                            "500.00MB",
+                            "1.00GB"]
+            actual_output = [rows[0][0],
+                            rows[0][1],
+                            rows[0][2],
+                            rows[0][3]]
+            self.assertEqual(expected_output, actual_output)
+
+        for val in ["True", "TRUE", "true"]:
```

Comment:
I would prefer to iterate through all kind of input and on that basis test should compare with expected outcome instead of repeating line#742-765 again.

As discussed offline please make sure to design test steps in future PRs that increases readability.